build-docker.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
```

Comment:
I do not like the file name `build-docker.sh`.
The reason is 
1. shell scripts in this project is alphabet (small letter) + underscore as a separator.
2. "docker" is product name. "containers" are already used as a output directory name for the purpose.

I prefer the file name `build_containers.sh`.

Reply:
This project's shell scripts are 4 spaces indent mainly. See removed `publish.sh` and removed lines in `test.sh`.
So, keep 4 spaces indent on shell scripts (*.sh).